lib/fetchers/local.rb

```diff
@@ -10,21 +10,29 @@ class Local < Inspec.fetcher(1)
     priority 0
 
     def self.resolve(target)
-      local_path = if target.is_a?(String)
-                     resolve_from_string(target)
-                   elsif target.is_a?(Hash)
-                     resolve_from_hash(target)
-                   end
-
-      new(local_path) if local_path
+      if target.is_a?(String)
+        local_path = resolve_from_string(target)
+        new(local_path) if local_path
+      elsif target.is_a?(Hash)
+        local_path = resolve_from_hash(target)
+        if local_path
+          # If target is a Hash then it is a dependency. We should archive it in
+          # order to create a checksum and vendor it. Unless the command used is
+          # exec then we should not vendor it to make local development easier.
+          target[:do_vendor] = true unless Inspec::BaseCLI.command == :exec
```

Comment:
Ahh. Inspec::BaseCLI.command is a hidden global.
Ideally, we'd add a param to the `resolve` interface, and pass along a flag saying whether to vendor the resolved profile.
Or, maybe we're OK with the global.

Reply:
@jquick and I couldn't find a clean way to pass it through to where we wanted. After much duress decided on a global variable.

If you can find any other way it would be seriously appreciated.